ui/media/js/main.js

```diff
@@ -712,6 +718,8 @@ async function checkTasks() {
 
     task.isProcessing = false
     task['stopTask'].innerHTML = '<i class="fa-solid fa-trash-can"></i> Remove'
+    task['useSettings'].innerHTML = '<i class="fa-solid fa-redo"></i> Use Settings'
```

Comment:
If this button's label will always be "Use Settings" (i.e. it never changes), we probably don't need this line right? Because the label is already set in line 853

Reply:
For Use Setting, my initial idea was to only show it once the task is completed. On second thought, I think I overdid it and should just keep it always visible, which also addresses your other two related comments.

Would that be better?

Reply:
Made the change. Use Settings is now always visible.